tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -43,9 +43,9 @@ ROOT::Experimental::RNTupleModel::RNTupleModel()
 ROOT::Experimental::RNTupleModel* ROOT::Experimental::RNTupleModel::Clone()
```

Comment:
Is the an advantage to having Clone being a non-const function?

Reply:
Addressed in #6655